configuration/parse-section.md

```diff
@@ -59,20 +59,20 @@ Here's the list of built-in parser plugins:
 
 ### Parse Parameters
 
+{% hint style='info' %}
+NOTE: It depends on parser implementation whether a plugin supports the following parse parameters. See each plugin's documentation in detail.
```

Comment:
Hmm, I thought we were trying to say that some input plugins do not support the `<parse>` section, so be careful.
Such as follows.

```suggestion
NOTE: It depends on a plugin (Input plugin in many cases) whether the plugin supports the `<parse>` section. See each plugin's documentation in detail.
```

The current sentences can be read as follows for me.
If my understanding is somehow incorrect, please let me know.

> It depends on the parser plugin whether the following parameters are supported.

Reply:
It may not wrong. So, probably, we need to clarify this sentence with more concrete description.

```suggestion
NOTE: It depends on an input plugin's capabilities whether can handle a `<parser>` section or not. See each plugin's documentation in detail.
```

Reply:
Thanks! I agree with the suggestion of @cosmo0920